core/src/test/resources/org/elasticsearch/action/fieldstats/fieldstats-index-constraints-request.json

```diff
@@ -28,6 +28,16 @@
       "max_value" : {
         "lt": 9
       }
+    },
+    "field1": {
```

Comment:
can we have one without a format? also, for a future change, can we not call them field 1 to 5 but something with meaning like date_field?